src/renderer/entities/proxy/lib/__tests__/mocks/proxy-mocks.ts

```diff
@@ -44,7 +38,7 @@ const wallets: Wallet[] = [
         accountId: TEST_ACCOUNTS[0],
         signingType: SigningType.MULTISIG,
         cryptoType: CryptoType.SR25519,
-      } satisfies VaultBaseAccount,
+      } satisfies BaseAccount,
```

Comment:
PV contains only 2 types of accounts - `VaultChainAccount` & `VaultShardAccount`